.asf.yaml

```diff
@@ -28,7 +28,21 @@ github:
     rebase:  true
 
   protected_branches:
-    main: { }
+    main:
+      required_status_checks:
+        # strict means "Require branches to be up to date before merging".
+        strict: false
+        # contexts are the names of checks that must pass
+        contexts:
+          - Scalafmt
+  
+      required_pull_request_reviews:
+        dismiss_stale_reviews: false
+        require_code_owner_reviews: false
+        required_approving_review_count: 1
```

Comment:
at least 2?

Reply:
only 1 review needed for trivial changes according to https://cwiki.apache.org/confluence/display/PEKKO/Processes - there is no way to work out automatically what is trivial

Reply:
So this is an interesting point. Historically akka always required 2 approvals, even for trivial changes. I will say however in terms of practicality especially right now with a lot of mechanical and trivial changes needed to her the project going 1 should be fine.

We can revisit this process later, I personally think that requiring 2 reviewers once the project is setup has merit especially since at that point in time we will also start seeing non trivial functional changes.

Reply:
Who is in the position to merge pull requests?

Reply:
anyone with git commit rights - all Pekko committers can do this - you might need to link your Github and ASF accounts if you haven't done this

Reply:
So the origin author and who proved the PR can merge it too?

Reply:
I don't know how to implement:

```
Any review that has open requests or discussion blocks the merge. (only merge when consensus is reached).
```

If there is a way, it can be implemented.

Reply:
> Any review that has open requests or discussion blocks the merge. (only merge when consensus is reached).

To me I think this is too strict, I have been in cases where pull requests stuck open in perpetuity because someone forgets to click the "resolve" button. Either that or committers will just blindly hit resolve which at that point defeats the purpose of it.

Should we discuss in the mailing list? I think for now this is fine, we can always add stricter checks later if there is a consensus on it.

Reply:
I would say lets just just merge the PR as it is now and we can also make a future one to implement these stricter checks.

Reply:
I don't see `required_conversation_resolution` in https://cwiki.apache.org/confluence/display/INFRA/git+-+.asf.yaml+features